src/commands.rs

```diff
@@ -111,11 +111,19 @@ pub fn mirror_question(
                     return Err(anyhow!("question has already resolved"));
                 }
             }
-            let question = (&kalshi_question)
-                .try_into()
-                .with_context(|| "failed to convert Kalshi question to common format")?;
-            let row = mirror::mirror_question(&client, &db, &question, config)?;
-            println!("Mirrored question:\n{:#?}", row);
+            match mirror::mirror_kalshi_question(&client, &db, config, &kalshi_question)
```

Comment:
Since we're only mirroring a single market here, it's fine for the function to just return an error on failure (using the `?` operator) and let the caller handle it.